server/src/main/scala/router.scala

```diff
@@ -382,7 +382,7 @@ object router {
         _            <- ZIO.logInfo(s"User '${token.user.details.email}' impersonated by ${impersonator.user.details.email}")
         impersonated = token.copy(impersonatedBy = Some(impersonator.user.details))
         encoded      <- tokens.encode(impersonated)
-      } yield loginResponse(impersonated, encoded)
+      } yield Response.json(encoded.toJson)
```

Comment:
The previous intent was to replace the morbid cookie with the new value. How are you planning to do that now?